### ISSUE TITLE
Choose picture size in Batch Manager

### ISSUE THREAD

**Issue by Thom1b** [Piwigo/Piwigo#579]
Hi,

First of all, sorry for my english.

I tag all my pictures with the name of people. I often have pictures with a group of people so I can't see them on the batch manager without zooming because batch manager only displays thumbnails.

It will be very very helpful for me if I can choose in batch manager the size of the pictures to display, just like we can already do when visiting an album.

Thanks to consider my request.

**Comment by plegall** [comment]
Hi @Thom1b 

Why don't you use the zoom feature on each thumbnail?

**Comment by Thom1b** [comment]
Hi,
I do, but it's fastidious to do that for each picture. It's very easier and less tiring for eyes (moins fatigant) for me to have not to click on each picture to see it.

**Comment by plegall** [comment]
You can also change the dimensions (in pixels) for the "thumbnail" size on page [Administration > Configuration > Options > Photo Sizes]. If you choose 235x235 pixels, it will be easier to see people faces without zooming.

**Comment by flop25** [comment]
I agree there is room for improvement for pictures/thumbnail sizes: album thumbnails, Batch Manager thumbnails, comments thumbnails etc

**Comment by Thom1b** [comment]
@plegall, I know I can do that, but I don't want my need to be imposed to visitors.

**Comment by hakangoker** [comment]
I agree we should be able to change size of the images displayed in Batch Manager. I had the same problem when I wanted to delete unnecessary 10k images.

Quick work around in batch_manager_global.tpl changing the image src from
$thumbnail.thumb->get_size_htm()
to
$thumbnail.FILE_SRC

THIS WILL BE OVERWRITTEN WHEN THERE WILL BE AN UPDATE BUT IT SAVED TIME FOR DELETING BULK IMAGES

**Comment by Thom1b** [comment]
Hi,
I gave up using piwigo since many years, I'm starting to use it again. As before I need to tag pictures with people name, as before it's a long job. I still have the same issue as before, thumbnails in batch manager in global view are so tiny I can't see who are on pictures. Of course I could zoom on each one but it's very easier if I could show bigger thumbnails.
I tried to change $thumbnail.thumb->get_size_htm() to $thumbnail.FILE_SRC like @hakangoker suggests but it doesn't work on 14.5.0.
If this feature still doesn't interest dev and people, I'll find a way to make it works.
This issue (really big for me) is the reason why I gave up piwigo.
Thanks!

**Comment by plegall** [comment]
We're preparing a face recognition plugin and a proper "people" manager (instead of using tags). That won't be for Piwigo 15, but for 16.

**Comment by Thom1b** [comment]
OK, so I understand my request still don't interest anybody. I hope I'll find a way quickly.
Thanks anyway!

**Comment by Thom1b** [comment]
It's really better for me (but not ideal) with this patch:

`diff -cr piwigo-14.5.0.orig/admin/themes/default/template/batch_manager_global.tpl piwigo-14.5.0/admin/themes/default/template/batch_manager_global.tpl
*** piwigo-14.5.0.orig/admin/themes/default/template/batch_manager_global.tpl	2024-07-15 17:09:05.000000000 +0200
--- piwigo-14.5.0/admin/themes/default/template/batch_manager_global.tpl	2024-07-19 17:18:50.241713333 +0200
***************
*** 606,612 ****
  						{if $thumbnail.level > 0}
  						<em class="levelIndicatorF" title="{'Who can see these photos?'|@translate} : ">{'Level %d'|@sprintf:$thumbnail.level|@translate}</em>
  						{/if}
! 						<img src="{$thumbnail.thumb->get_url()}" alt="{$thumbnail.file}" title="{$thumbnail.TITLE|@escape:'html'}" {$thumbnail.thumb->get_size_htm()}>
  					</span>
  				</label>
  			</span>
--- 606,612 ----
  						{if $thumbnail.level > 0}
  						<em class="levelIndicatorF" title="{'Who can see these photos?'|@translate} : ">{'Level %d'|@sprintf:$thumbnail.level|@translate}</em>
  						{/if}
! 						<img src="{$thumbnail.FILE_SRC}" alt="{$thumbnail.file}" title="{$thumbnail.TITLE|@escape:'html'}" {$thumbnail.thumb->get_size_htm()}>
  					</span>
  				</label>
  			</span>
diff -cr piwigo-14.5.0.orig/admin/themes/default/theme.css piwigo-14.5.0/admin/themes/default/theme.css
*** piwigo-14.5.0.orig/admin/themes/default/theme.css	2024-07-15 17:09:05.000000000 +0200
--- piwigo-14.5.0/admin/themes/default/theme.css	2024-07-19 17:17:39.185914984 +0200
***************
*** 6242,6248 ****
  	background-color: transparent; padding: 0; box-shadow: none; font-size: initial; margin-bottom: 0;
  }
  #batchManagerGlobal UL.thumbnails .wrap2 {width: 110px !important; height: 110px !important;}
! #batchManagerGlobal UL.thumbnails IMG {margin-top: 0; width: 110px; height: 110px; border: 1px solid transparent;}
  #batchManagerGlobal UL.thumbnails .wrap2 .icon-pencil {font-size: 15px; margin-left: 4px;}
  #batchManagerGlobal UL.thumbnails .wrap2 .icon-plus {margin-right: 2px;}
  #batchManagerGlobal li .font-checkbox [class*="icon-check"] {height: 20px; width: 20px; border-radius: 100%}
--- 6242,6248 ----
  	background-color: transparent; padding: 0; box-shadow: none; font-size: initial; margin-bottom: 0;
  }
  #batchManagerGlobal UL.thumbnails .wrap2 {width: 110px !important; height: 110px !important;}
! #batchManagerGlobal UL.thumbnails IMG {margin-top: 0; width: 300px; height: 300px; border: 1px solid transparent;}
  #batchManagerGlobal UL.thumbnails .wrap2 .icon-pencil {font-size: 15px; margin-left: 4px;}
  #batchManagerGlobal UL.thumbnails .wrap2 .icon-plus {margin-right: 2px;}
  #batchManagerGlobal li .font-checkbox [class*="icon-check"] {height: 20px; width: 20px; border-radius: 100%}
***************
*** 7184,7187 ****
    .promote-content .left-side img {
      width: 100px;
    }
! }
\ Pas de fin de ligne à la fin du fichier
--- 7184,7187 ----
    .promote-content .left-side img {
      width: 100px;
    }
! }
`
